dv_utils/settings.py

```diff
@@ -33,8 +33,8 @@ def load_settings(self, dotenv_file: str = None):
 
         self.base_url: str = config("DV_URL", default="", cast=str)
         self.token: str = config("DV_TOKEN", default="", cast=str)
-        self.app_id: str = config("DV_APP_ID", default="", cast=str)
-        self.client_id: str = config("DV_CLIENT_ID", default="", cast=str)
+        self.collaboration_space_id: str = config("DV_COLLABORATION_SPACE_ID", default="", cast=str)
```

Comment:
I don't think this environment variable is injected in the cage. Do you have any particular reason to change this?

Reply:
Need to discuss